Presentation.Web/Partials/org/user/org-user.html

```diff
@@ -12,30 +12,30 @@ <h3 class="panel-title">
         </h3>
     </div>
 
-    <table class="table table-condensed table-striped table-fixed">
+    <table class="table table-condensed table-striped tablehttps://localhost:44300/contract/overview-fixed">
         <thead>
             <tr>
-                <th class="col-md-2">
+                <th width="200">
```

Comment:
Jeg så hellere du lavede en CSS klasse istedet for at "hardcode" det.
Men allerhelst at det blev fixet med eksisterende bootstrap classes